src/components/renderer/form-masked-input.vue

```diff
@@ -116,7 +119,7 @@ export default {
     getMask() {
       const format = this.dataFormat === 'date' ? this.getUserDateFormat() : this.getUserDateTimeFormat();
       return typeof masks[format] === 'undefined'
-        ? masks['default']
+        ? defaultMask()
```

Comment:
Why does `defaultMask` need to be a function? Is it because the mask is modified and so a new object reference is required in each case?

Reply:
Fair point